src/languages/en.ts

```diff
@@ -39,6 +39,7 @@ import type {
     ChangeTypeParams,
     CharacterLengthLimitParams,
     CharacterLimitParams,
+    ChatWithAMParams,
```

Comment:
I would prefer if this was `ChatWithAccountManagerParams` and everywhere else we do `am` changed to `accountManager`

Reply:
@marcaaron I updated.